JSDemos/Demos/Charts/SideBySideStackedBar/description.md

```diff
@@ -1 +1,23 @@
-In&nbsp;this example, side-by-side mode for the _stacked bar_ series type is&nbsp;used to&nbsp;group bar series based on&nbsp;their **stack** property value. Bar series with common characteristics are displayed and stacked in&nbsp;one column.
\ No newline at end of file
+This demo shows the _side-by-side stacked bar_ series that allows you to visualize data as bars stacked in different groups. This series type is useful when you compare an individual series' values with groups' total values.
+
+## Bind to Data
+
+In this demo, series is bound to data directly. See the [Bind Series to Data](/Documentation/Guide/UI_Components/Chart/Data_Binding/Bind_Series_to_Data) article for information. The "state" field name is assigned to the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/#argumentField) property of the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object since the side-by-side stacked bar chart contains series with the same argument field. The side-by-side stacked bar series is grouped based on the [stack](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#stack) property value. 
+
+## Customize Side-By-Side Stacked Bar Chart
+
+### Customize Chart Legend 
+
+Use the [verticalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#verticalAlignment) and [horizontalAlignment](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#horizontalAlignment) properties of the legend object to specify the legend layout. 
+
+The [customizeLegend](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#customizeItems) function allows you to change the order, text, and visibility of legend items. In this demo, the **array.splice** method is used in the **customizeItems** function to sort the legend items. If you want to customize the legend's border, use properties collected in the [border](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/border) object. 
+
+To distribute all legend items between several columns (in a vertically-oriented legend) or rows (in a horizontally-oriented legend), set the [columnCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#columnCount) or [rowCount](/Documentation/ApiReference/UI_Components/dxChart/Configuration/legend/#rowCount) property respectively.
```

Comment:
set the properties to what?

Reply:
@arminal  Let's change set to "specify" the properties

Reply:
yes